pkg/api/graph/graph.go

```diff
@@ -636,6 +637,24 @@ func NodesByKind(g Interface, nodes []graph.Node, kinds ...string) [][]graph.Nod
 	return result
 }
 
+// IsFromDifferentNamespace returns if a node is in a different namespace
+// than the one provided.
+func IsFromDifferentNamespace(namespace string, node graph.Node) bool {
```

Comment:
This sounds picky, but I guarantee that a re-user of this method won't expect the `Found` behavior.  Can you either separate the tests (`IsInNamespace` and `IsFound`) or add the detail in the name?